### PR TITLE
[Teva 2384] create pentest application environment

### PR DESCRIPTION
**Pull request by felixtheflex**
## Jira ticket URL

- Just add the ticket number to the end:

https://dfedigital.atlassian.net/browse/TEVA-2384

## Changes in this PR:

create pentest application environment
 -

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3265.london.cloudapps.digital>

**Review by csutter — APPROVED** [review]
:+1:

One tiny change I forgot about – could you set `AUTHENTICATION_FALLBACK` to `true` in the env?

File: terraform/workspace-variables/pentest.tfvars [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Platform
+region                      = "eu-west-2"
+environment                 = "pentest"
+app_environment             = "pentest"
+parameter_store_environment = "qa"
+
+# CloudFront
+distribution_list = {
+  "tvsqa" = {
```
Comment (saliceti) [inline terraform/workspace-variables/pentest.tfvars:9]: This should be called tvspentest to avoid confusion

Comment (felixtheflex) [reply terraform/workspace-variables/pentest.tfvars]: done

**Review by saliceti — APPROVED** [review]

**Pull request merged**